src/Adapter/Module/ModuleDataProvider.php

```diff
@@ -219,6 +219,10 @@ public function isModuleMainClassValid($name)
         }
 
         $parser = (new PhpParser\ParserFactory())->create(PhpParser\ParserFactory::PREFER_PHP7);
+        $log_context_data = [
+            'object_type' => 'Module',
+            'object_id' => LegacyModule::getModuleIdByName($name),
+         ];
 
         try {
```

Comment:
```suggestion
        $log_context_data = [
            'object_type' => 'Module',
            'object_id' => LegacyModule::getModuleIdByName($name),
        ];

        try {
```

Weird that php-cs-fixer is not crying oO